book.rb

```diff
@@ -0,0 +1,14 @@
+# book.rb
+class Book
+  attr_accessor :title, :author
```

Comment:
- [ ] kindy make sure that from the book we can get all the rentals, and to do this you need to make the rentals array to be accessed outside the Book.
```suggestion
  attr_accessor :title, :author, :rentals
```